backend/uclapi/resources/views.py

```diff
@@ -0,0 +1,52 @@
+import os
+import requests
+
+from lxml import etree
+
+from common.decorators import uclapi_protected_endpoint
+from common.helpers import PrettyJsonResponse as JsonResponse
+
+from rest_framework.decorators import api_view
+
+
+@api_view(['GET'])
+@uclapi_protected_endpoint()
+def get_pc_availability(request, *args, **kwargs):
+    try:
+        r = requests.get(os.environ["PCA_LINK"])
+    except requests.exceptions.MissingSchema:
+        return JsonResponse({
+            "ok": False,
+            "error": "Couldn't get the availability data. Try again later"
```

Comment:
Full stop after error? We should try to be coordinated with our messages and make them more formal, for example:
`Could not retrieve availability data. Please try again later or contact us for support.`